### PR TITLE
Add DNC.org.xml and Democrats.org.xml

### PR DESCRIPTION
**Pull request by jeremyn**

### CODE REVIEW

File: src/chrome/content/rules/DNC.org.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--
+	See also Democrats.org.xml.
+
```
Comment (J0WI) [inline src/chrome/content/rules/DNC.org.xml]: See also:
hrb.dnc.org
gems.dnc.org
sendgrid.dnc.org
tanto.dnc.org

File: src/chrome/content/rules/Democrats.org.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		- centrodevotacion.democrats.org
+		- convention.democrats.org
+		- ppapi.democrats.org
+
+	Time out:
+		- asdc.democrats.org
+		- developers.democrats.org
+		- directory.democrats.org
+		- factivists.democrats.org
+
```
Comment (J0WI) [inline src/chrome/content/rules/Democrats.org.xml]: See also:
developers.democrats.org
dtmc.democrats.org
future.democrats.org
live.democrats.org

Comment (jeremyn) [inline src/chrome/content/rules/Democrats.org.xml]: https://developers.democrats.org is already in the file so I'm not sure what you want me to do with that subdomain.

File: src/chrome/content/rules/DNC.org.xml (hunk 1 above)
Comment (jeremyn) [inline src/chrome/content/rules/DNC.org.xml]: Both https://gems.dnc.org and the HTTP version time out for me, so I don't think we should include or even mention it.

**Pull request merged**